markdown/docs/user-guide.md

```diff
@@ -28,370 +37,455 @@ You can start a FastBoot server on your development machine by running:
 ember fastboot
 ```
 
-This starts a FastBoot server listening on port 3000. You can verify
-it's working by curling from `localhost`:
+This starts a FastBoot server listening on port 3000\.
+You can verify it's working by curling from `localhost`:
 
 ```sh
 curl http://localhost:3000
 ```
 
+You should see your index route fully rendered (as opposed to the skeletal `index.html` page Ember usually sends).
+
 To stop the server, press `Ctrl-C` on your keyboard to kill the process.
 
-Note that, at the moment, the server does not automatically restart when
-you make changes. If you make changes to your application, you'll need
-to kill the server and restart it.
+Note that, at the moment, the server does not automatically restart when you make changes.
```

Comment:
Let's remember to remove this if @stefanpenner is able to fix the watch-related bugs. :)